changelogs/fragments/450-slackpkg-package-matching.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
+ - slackpkg - Fix name matching in package installation (https://github.com/ansible-collections/community.general/issues/450).
```

```suggestion
 - slackpkg - fix name matching in package installation (https://github.com/ansible-collections/community.general/issues/450).
```
https://docs.ansible.com/ansible/devel/community/development_process.html#changelogs-how-to the latest doc says `start the entry with - [module name] - and the following sentence with a lowercase letter.`